pkg/configutil/once.go

```diff
@@ -24,6 +24,10 @@ func LoadConfigOnce() (*config.Config, error) {
 	var err error
 	configOnce.Do(func() {
 		configInfo, err = config.LoadConfig()
+		// set default value
+		if configInfo.EnvelopeType == "" {
+			configInfo.EnvelopeType = "jws"
```

Comment:
I found that we can use `JwsFormat` in package `cmd` instead of hard code a string. But it will introduce a cycle import. Do you think we should move `JwsFormat` and `CoseFormat` to another package in the `internal` (Maybe we can create an `internal/envelope`) so that both `cmd` and `configutil` can import them.
Then your code will be
```
import "github.com/notaryproject/notation/internal/evnelope"
// ...
if configInfo.EnvelopeType == "" {
    configInfo.EnvelopeType = envelope.Jws
}
```

Reply:
Good idea! Thanks.